app/views/layouts/application.html.erb

```diff
@@ -7,7 +7,7 @@
     <%= csp_meta_tag %>
 
   <link rel="stylesheet" href="https://rsms.me/inter/inter.css">
-    <%= stylesheet_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_pack_tag 'shops', 'data-turbolinks-track': 'reload' %>
```

Comment:
???
you remove here all our tailwind stylesheets and components

Reply:
Sorry, some accident. Done